### ISSUE TITLE
multiline edits

### ISSUE THREAD

**Issue by sweirich** [antalsz/hs-to-coq#32]
Making each edit take exactly one line is harsh for redefine / add edits.   

Can we add a multiline-block to the parser so that instead of 
```
redefine Local Definition instance_GHC_Base_Eq__Boxity_op_zsze__ : Boxity -> (Boxity -> bool) := fun arg_319__ arg_320__ =>  match arg_319__ , arg_320__ with | a , b => negb (instance_GHC_Base_Eq__Boxity_op_zeze__ a b)  end.  
```

we can write 

```
redefine {{ Local Definition instance_GHC_Base_Eq__Boxity_op_zsze__ : Boxity -> (Boxity -> bool) := 
    fun arg_319__ arg_320__ =>  
        match arg_319__ , arg_320__ with 
            | a , b => negb (instance_GHC_Base_Eq__Boxity_op_zeze__ a b)  
        end. 
}}
```

**Comment by nomeata** [comment]
Or we could just require a `.` at the end of every edit, instead of looking for newlines. Might be more convenient for the Coq-experience user.

**Comment by antalsz** [comment]
I think the happy medium there is to treat newlines as whitespace *iff* we're in a Coq definition, and require a period there.  (I really want to preserve the newlineless syntax for simple things like `skip` and `rename`.)

**Comment by antalsz** [comment]
Fixed by bfb586488fa529b829b8d16659ef1c4f52b25c6e – all Coq definitions in the edits file can have embedded newlines and must end with a `.`.
